addons/website_profile/controllers/main.py

```diff
@@ -142,20 +126,16 @@ def _profile_edition_preprocess_values(self, user, **kwargs):
             values['website_published'] = kwargs.get('website_published') == 'True'
         return values
 
-    @http.route('/profile/user/save', type='http', auth="user", methods=['POST'], website=True)
-    def save_edited_profile(self, **kwargs):
-        user_id = int(kwargs.get('user_id', 0))
+    @http.route('/profile/user/<int:user_id>', type='http', auth="user", methods=['POST'], website=True)
```

Comment:
Is it necessary to have the same endpoint with GET/POST methods for this?

It's rather unusual I think, a more common option would be "/profile/user/<int:user_id>/save" or something similar.